mautic/mautic
# Fix display Google Analytics code in landing page

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/9352




#### Description:
Google analytics code was converted to boolean type and not displayed on landing page.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to Configurations and add some HTML to Analytics script under Landing Page Settings
3. Save
4. Check local.php a and see google_analytics parameter was saved as 1
5. After PR works properly
